packages/apps/src/plugins/http/plugin.ts

```diff
@@ -217,15 +233,32 @@ export class HttpPlugin implements ISender {
     }
 
     const activity: Activity = req.body;
-    const token: IToken = authorization
-      ? new JsonWebToken(authorization)
-      : {
-          appId: '',
-          from: 'azure',
-          fromId: '',
-          serviceUrl: activity.serviceUrl || '',
-          isExpired: () => false,
-        };
+    let token: IToken;
+    if (this.botTokenValidator) {
+      if (!authorization) {
```

Comment:
this check is already implemented in line 230.

Reply:
Technically only accurate if node_env !== "local". TS complains otherwise